Core/VGPlib.c

```diff
@@ -569,8 +570,8 @@ static void readStringList(VgpFile *vf, char t, I64 len)
   free (string);
 }
 
-BOOL vgpReadLine (VgpFile *vf)
-{ BOOL      isAscii;
+char vgpReadLine (VgpFile *vf)
```

Comment:
This was one location where I think `char` is preferred. 

Previously from `typedef char BOOL;`